inspirehep/migrator/tasks.py

```diff
@@ -278,3 +346,11 @@ def migrate_record_from_mirror(prod_record):
         prod_record.valid = True
         db.session.merge(prod_record)
         return record
+
+
+def wait_for_all_tasks(task):
+    print(f"Waiting for {task}.")
```

Comment:
Why `print`? maybe better to use a logger if you think you need to log this.

Reply:
Logger is not logging stuff to stdout when you run cli command.

Reply:
currently, we are using invenio-logging which means we have to set the level for ``stdout`` like this https://github.com/inveniosoftware/invenio-logging/blob/master/invenio_logging/config.py#L53

Reply:
but we will merge soon https://github.com/inspirehep/inspirehep/pull/279 and drop invenio-logging. Whu not  `click.echo`?

Reply:
why guys we merged it like this? Using print seems small but still IMO doesn't make any sense using it here